src/components/Status/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Theme, css } from '@emotion/react'
+import styled from '@emotion/styled'
+import React from 'react'
+import { flash } from '../../utils'
+import Tooltip from '../Tooltip'
+
+const variants = {
+  danger: ({ theme }: { theme: Theme }) => `
+      background-color: ${theme.colors.danger.backgroundStrong};
+    `,
+  info: ({ theme }: { theme: Theme }) => `
+      background-color: ${theme.colors.info.backgroundStrong};
+    `,
+  neutral: ({ theme }: { theme: Theme }) => `
+      background-color: ${theme.colors.neutral.backgroundStronger};
+    `,
+  success: ({ theme }: { theme: Theme }) => `
+      background-color: ${theme.colors.success.backgroundStrong};
+    `,
+  warning: ({ theme }: { theme: Theme }) => `
+      background-color: ${theme.colors.warning.backgroundStrong};
+    `,
+} as const
+type StatusVariant = keyof typeof variants
+export const statusVariants = Object.keys(variants) as StatusVariant[]
+
+const cssAnimation = css`
+  animation: ${flash} linear 1s infinite;
+`
+
+type StyledCircleProps = Pick<StatusProps, 'variant'>
+const StyledCircle = styled('div', {
+  shouldForwardProp: prop => prop !== 'variant',
+})<StyledCircleProps>`
+  display: inline-block;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  ${({ variant }) => variants[variant]}
+`
+
+type StatusProps = {
+  animated?: boolean
+  className?: string
+  variant: StatusVariant
+  tooltip?: string
+}
+
+const Status = ({
+  animated = false,
+  className,
+  tooltip,
+  variant,
+}: StatusProps): JSX.Element => (
+  <Tooltip text={tooltip}>
+    <StyledCircle
+      className={className}
+      css={animated && cssAnimation}
+      variant={variant}
+    />
```

Comment:
Can't we pass animated prop directly to `StyledCircle` and condition with ternary `animation: ${flash} linear 1s infinite;` there directly ?

Reply:
done